content/spin/kv-store.md

```diff
@@ -0,0 +1,355 @@
+title = "Using Key Value Storage with Spin Applications"
+template = "spin_main"
+date = "2023-02-21T00:00:00Z"
+enable_shortcodes = true
+[extra]
+
+---
+- [Using Key Value Storage With Spin Applications](#using-key-value-storage-with-spin-applications)
+    - [Prerequisites](#prerequisites)
+    - [Use Spin SDK To Open And Persist Data In A Default Key/Value Store](#use-spin-sdk-to-open-and-persist-data-in-a-default-keyvalue-store)
+    - [Configure `spin.toml` To Use A Default Key/Value Store](#configure-spintoml-to-use-a-default-keyvalue-store)
+    - [Build And Deploy Your Spin App](#build-and-deploy-your-spin-app)
+    - [Store and Retrieve Data From Your Default Key/Value Store](#store-and-retrieve-data-from-your-default-keyvalue-store)
+    - [Conclusion](#conclusion)
+    - [Next Steps](#next-steps)
+
+## Using Key Value Storage with Spin Applications
+
+ Spin application are best suited for event-driven, stateless workloads that have low-latency requirements. If your Spin applications needs to persist state, the data will need to be stored in an external tool or service. In [Spin v0.9.0](https://www.fermyon.com/blog/spin-v09), we've added support for key/value storage. This is a storage option for applications that need to data in the form of key/value pairs and are satistifed by a BASE consistency model. Workload examples include general value caching, session caching, counters, serialized application state.
+
+ Spin users can use new a API in the Spin SDKs for persisting and retrieving non-relational data from a key/value store across multiple requests to and from the same application. Together with the updated SDKs we are introducing a built-in, _local_ key/value store available with minimal configuration for every Spin application. The default key/value store is available for the following SKDs: Go, JavaScript/TypeScript, and Rust. 
```

Comment:
These sentences are a little long. I'd break them up more, maybe like the following:
```suggestion
 This new key/value API is available to Spin users via various language SDKs. Now spin users can persist and retrieve non-relational data from a key/value store across multiple requests to and from the same application. Along with the updated SDKs, we are introducing a built-in, _local_ key/value SQLite database available with minimal configuration for every Spin application. The default key/value store is available for the following SKDs: Go, JavaScript/TypeScript, and Rust. 
```